docs/sources/developer/Dependencies.md

```diff
@@ -1,48 +1,36 @@
 # Plaso Dependencies
```

Comment:
I opt: Plaso development dependencies

and maybe rename the file to "Development-Dependencies.md" or equiv.

Reply:
Done